app/MMB/src/scenes/explore/ExploreBookingOneWay.js

```diff
@@ -0,0 +1,176 @@
+// @flow
+
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+import { TextIcon } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { TitledMenuGroup, MenuItem } from '@kiwicom/mobile-navigation';
+import idx from 'idx';
+
+import type { ExploreBookingOneWay as ExploreBookingOneWayType } from './__generated__/ExploreBookingOneWay.graphql';
+
+type Props = {|
+  +data: ExploreBookingOneWayType,
+|};
+
+const warnTodo = () => console.warn('TODO');
+
+const renderItem = (isAirport: boolean, title: string, description: string) => (
+  <MenuItem
+    title={<Translation passThrough={title} />}
+    description={<Translation passThrough={description} />}
+    icon={isAirport ? <TextIcon code="a" /> : <TextIcon code="c" />}
+    isActive={false}
+    onPress={warnTodo}
+  />
+);
```

Comment:
This looks like the exact same renderItem as above, extract it to it's own file, and reuse it from both components